partiql-parser/src/parse/partiql.lalrpop

```diff
@@ -1251,6 +1315,24 @@ CommaTermPlus<T>: Vec<T> = {
     }
 }
 
+// Comma as terminator (i.e. "<T>, <T>, <T>," where final comma is optional); may be empty
+// This is a macro; see http://lalrpop.github.io/lalrpop/tutorial/006_macros.html
+SynthCommaTermStar<T>: Synth<Vec<T>> = {
+    <v:(<T> ",")*> <e:T?> => match e {
+        None => v.into_iter().collect(),
+        Some(e) => v.into_iter().chain( std::iter::once(e) ).collect(),
+    }
+}
+
+// Comma as terminator (i.e. "<T>, <T>, <T>," where final comma is optional); at least 1 arg
+// This is a macro; see http://lalrpop.github.io/lalrpop/tutorial/006_macros.html
+SynthCommaTermPlus<T>: Synth<Vec<T>> = {
+    <v:(<T> ",")+> <e:T?> => match e {
+        None => v.into_iter().collect(),
+        Some(e) => v.into_iter().chain( std::iter::once(e) ).collect(),
+    }
+}
```

Comment:
I'm still reading up on lalrpop macros but these don't seem to be used in the grammar?